src/ipfs/util.ts

```diff
@@ -1,19 +1,19 @@
-import dagPB from 'ipld-dag-pb'
-import { DAGNode, RawDAGNode, DAGLink, RawDAGLink } from './types'
+import dagPB, { DAGLink, DAGNode } from 'ipld-dag-pb'
+import type { GetResult } from 'ipfs-core-types/src/dag'
+import { CID } from 'ipfs-message-port-client/src/block'
```

Comment:
Is this the same `CID` type as from the `cids` library?

Reply:
Nope. Super annoying that it isn't. Not sure what's up with that in the js-ipfs ecosystems right now :thinking: